rednotebook/info.py

```diff
@@ -431,6 +431,13 @@ def get_commandline_parser():
 you can see the errors right away and get rid of them by editing the
 file.
 
+If you want to use the Latex index you need to run
+
+``makeindex your-rednotebook-export.idx``
+
+after you first built the PDF file. After this you need to build the PDF
+again.
```

Comment:
Let's just write the following instead:

    Use the following commands to include an index of tags in the PDF: 
    ```
    pdflatex your-rednotebook-export.tex
    makeindex your-rednotebook-export
    pdflatex your-rednotebook-export.tex
    ```